pkg/trace/pb/README.md

```diff
@@ -0,0 +1,15 @@
+## Generating Go files from protobuf files
```

Comment:
I'm on the fence about this. Some of the generated code might've been modified. I'm not sure if we want to encourage folk to re-generate this.

Reply:
Ok, I think the best thing would be to have some sort of makefile to generate all that (with the modifications). But it has nothing to do with that PR. I will remove it for now.